### PR TITLE
black: add --skip-string-normalization example

### PR DESCRIPTION
**Pull request by naltun**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Add a `black --skip-string-normalization` example. This is useful for applying the `black` formatting to codebases that wish to maintain single quotes. By default, `black` will change `'` to `"`.

E: remove short option `-S`.

### CODE REVIEW

File: pages/common/black.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Auto-format a file or directory emitting exclusively error messages to stderr:
 
 `black --quiet {{path/to/file_or_directory}}`
+
+- Auto-format a file or directory without replacing single quotes with double quotes:
+
+`black -S {{path/to/file_or_directory}}`
```
Comment (navarroaxel) [inline pages/common/black.md:28]: We prefer long arguments when are available:
```suggestion
`black --skip-string-normalization {{path/to/file_or_directory}}`
```

File: pages.ko/common/black.md [hunk 2]
```diff
@@ -22,3 +22,7 @@
 - 파일 또는 디렉토리가 stderr에 배타적 오류 메시지를 발생시키는 자동 포맷:
 
 `black --quiet {{파일_또는_디렉토리/의/경로}}`
+
+- 작은 따옴표를 큰 따옴표로 바꾸지 않고 파일 또는 디렉토리 자동 서식 지정:
+
+`black -S {{파일_또는_디렉토리/의/경로}}`
```
Comment (navarroaxel) [inline pages.ko/common/black.md:28]:
```suggestion
`black --skip-string-normalization {{파일_또는_디렉토리/의/경로}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @naltun, thanks for your submission. I left one easy comment.

File: pages/common/black.md (hunk 1 above)
Comment (naltun) [reply pages/common/black.md:28]: @navarroaxel Perfect. I do, too. Should I update the existing short argument, etc. `black -c => black --code`?

File: pages.ko/common/black.md (hunk 2 above)
Comment (naltun) [reply pages.ko/common/black.md]: Done in 49b7216.

File: pages/common/black.md (hunk 1 above)
Comment (naltun) [reply pages/common/black.md]: Done in 49b7216.

**Review by sbrl — APPROVED** [review]
Thanks, @naltun!

**Review by navarroaxel — APPROVED** [review]

File: pages/common/black.md [hunk 3]
```diff
@@ -22,3 +22,7 @@
 - Auto-format a file or directory emitting exclusively error messages to stderr:
 
 `black --quiet {{path/to/file_or_directory}}`
+
+- Auto-format a file or directory without replacing single quotes with double quotes:
```
Comment (nicokosi) [inline pages/common/black.md:26]: Should we mention about [its use case from the documentation](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings)?
> adoption helper, avoid using this for new projects

Maybe like this?
> Auto-format a file or directory without replacing single quotes with double quotes (adoption helper, avoid using this for new projects)

Comment (sbrl) [reply pages/common/black.md:26]: Hmmm, I'm not sure. We want to make sure the description isn't too long

Comment (marchersimon) [reply pages/common/black.md:26]: I'd include it. The description isn't too long and this sounds like important info.

Comment (naltun) [reply pages/common/black.md]: Hi @marchersimon, please review 0133152.

**Comment by CleanMachine1** [comment]
Thanks for your contribution.

**Pull request merged**